lib/decidim/participatory_documents/admin_engine.rb

```diff
@@ -15,6 +15,9 @@ class AdminEngine < ::Rails::Engine
             get :pdf_viewer
             get :edit_pdf
           end
+          member do
+            post :publish
```

Comment:
```suggestion
            put :publish
```
Let's use put here for consistency with the rest of Decidim